src/statistiek_hub/utils/check_import_fields.py

```diff
@@ -0,0 +1,18 @@
+
+def check_missing_import_fields( fields: list, expected:list):
+    """ returns 
+            -error: if not all expected items exist in fields
+            -false if all exist
+    """
+
+    # check list items
+    list_a = fields
+    list_b = expected
+
+    _diff = list(set(list_b) - set(list_a)) 
+
+    if len(_diff) > 0:
```

Comment:
Een lege `set()` is falsy, en een niet-lege `set()` is truthy. Volgens mij zou dit genoeg moeten zijn:

```
_diff = set(expected) - set(fields)
if _diff:
   ...
else:
   ...
```

Op deze manier is het wel zo dat de functie een truthy waarde teruggeeft bij een foutmelding, en een falsy waarde wanneer alles goed is. Misschien een kwestie van smaak, maar voor zou het logischer zijn als dat andersom was. Je zou ook nog kunnen overwegen om te `raise`en bij een error, dan is het helemaal duidelijk.